mock/py/mockbuild/util.py

```diff
@@ -425,6 +425,13 @@ def unshare(flags):
     except AttributeError:
         pass
 
+    if flags & CLONE_NEWNS:
+        # Unsharing the mount namespace isn't immediately effective if the
+        # source mount propagation status is 'shared' - changes to the mounts
+        # will still propagate back to the parent namespace. Do the same
+        # thing as unshare(1) and make all mounts private.
+        do(['mount', '--make-rprivate', '/'])
```

Comment:
This will change the flag on / of host (!) - I am not sure we want this.

Reply:
Are you sure? It's after the call to unshare()

Reply:
Yes, I am sure. Try to replace it with `['touch', '/DBG']` and see where the file will be created.